src/txmempool.cpp

```diff
@@ -465,16 +678,22 @@ void CTxMemPool::remove(const CTransactionBase& origTx, std::list<CTransaction>&
             for(const CTxIn& txin: cert.GetVin())
                 mapNextTx.erase(txin.prevout);
 
+            const uint256& scid = cert.GetScId();
```

Comment:
This block could be set under ifdef as well, isTopQualityCert is used only below